minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-framework-core/src/main/java/org/apache/nifi/minifi/status/StatusRequestParser.java

```diff
@@ -122,15 +122,13 @@ static RemoteProcessGroupStatusBean parseRemoteProcessGroupStatusRequest(RemoteP
                 new BulletinQuery.Builder()
                         .sourceIdMatches(inputRemoteProcessGroupStatus.getId())
                         .build());
-        List<String> authorizationIssues = inputRemoteProcessGroupStatus.getAuthorizationIssues();
```

Comment:
Ah ok, could you remove "authorizationIssues" as a query option from the docs then?